client/src/helpers/stardust/transactionsHelper.ts

```diff
@@ -163,6 +160,12 @@ export class TransactionsHelper {
         return { inputs, outputs: [...outputs, ...remainderOutputs], unlockAddresses, transferTotal };
     }
 
+    public static getTransactionId(payload: ITransactionPayload) {
```

Comment:
Let's call this `computeTransactionIdFromTransactionPayload` ? Wdyt ?

Reply:
Yeah sounds better. Done.